index.ts

```diff
@@ -62,7 +63,7 @@ function useReplicantRaw<T>(name: string, namespace: string | ReplicantOptions<T
         newVal.value = clone(oldVal.value)
     }
     function loadDefault() {
-        newVal.value = clone(opts?.defaultValue)
+        newVal.value = clone(opts && "defaultValue" in opts ? opts.defaultValue : undefined)
```

Comment:
I'm might be missing something but I'm not sure why this is needed? Isn't `defaultValue` just gonna be `undefined` if it isn't `in opts`?

Reply:
No, because of how the types work, it will just outright refuse it because it doesn't see `defaultValue` as just being possibly defined, but not possible at all in some situations, hence this check.

Reply:
Understood, thanks for clarifying.